src/analysis/normed_space/basic.lean

```diff
@@ -1081,14 +1081,33 @@ lemma has_sum_iff_tendsto_nat_of_summable_norm {f : ℕ → α} {a : α} (hf : s
 ⟨λ h, h.tendsto_sum_nat,
 λ h, has_sum_of_subseq_of_summable hf tendsto_finset_range h⟩
 
-variable [complete_space α]
-
 /-- The direct comparison test for series:  if the norm of `f` is bounded by a real function `g`
 which is summable, then `f` is summable. -/
-lemma summable_of_norm_bounded {f : ι → α} (g : ι → ℝ) (hg : summable g) (h : ∀i, ∥f i∥ ≤ g i) :
+lemma summable_of_norm_bounded
+  [complete_space α] {f : ι → α} (g : ι → ℝ) (hg : summable g) (h : ∀i, ∥f i∥ ≤ g i) :
   summable f :=
 by { rw summable_iff_cauchy_seq_finset, exact cauchy_seq_finset_of_norm_bounded g hg h }
 
+/-- If the sum of `f` is known to exist by the direct comparison test, then its norm is bounded by
+that of the comparator. -/
```

Comment:
The docstring is a little bit misleading, as the lemma does not claim that the sum of `f` exists. If it doesn't, it's just that the lemma is also true because the left hand side is `0`.